include_core/omrgcconsts.h

```diff
@@ -423,7 +423,8 @@ typedef enum {
 	SCAV_RATIO_TOO_LOW,
 	HEAP_RESIZE,
 	SATISFY_EXPAND,
-	FORCED_NURSERY_CONTRACT
+	FORCED_NURSERY_CONTRACT,
+	SOFT_MX_CONTRACT,
```

Comment:
I just realized that https://github.com/eclipse/omr/pull/5825 introduces almost identical flag, called `SATISFY_SOFT_MX`.

Perhaps it would be better to re-use the same flag? @amicic

Reply:
right, should be just one. 
keep them in both PRs and whoever goes first will introduce it, the second will reuse the one for first PR